src/ItemList.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { db } from './lib/firebase';
+import { useCollection } from 'react-firebase-hooks/firestore';
+
+function ItemList() {
+  const [value, loading, error] = useCollection(
+    db.collection('randomTokenOne'),
+    {
+      snapshotListenOptions: { includeMetadataChanges: true },
+    },
+  );
+
+  return (
+    <div>
+      {error && <strong>Error: {JSON.stringify(error)}</strong>}
+      {loading && <span>Collection: Loading...</span>}
+      {value && (
+        <>
+          <h2>Shopping List:</h2>
+          <ul>
+            {value.docs.map((doc) => (
+              <li key={doc.id}>
+                {
+                  JSON.stringify(doc.data()['formData']['itemName']).replace(
+                    /['"]+/g,
+                    '',
+                  ).toca
+                }
```

Comment:
It might be nice to just display the name:
![image](https://user-images.githubusercontent.com/10378331/114869745-f719ae80-9dbc-11eb-9677-00590d11cf1c.png)

Reply:
Omg, I left the remnants of a random method (ie. the `.toca`) there 😅. That's likely why the names weren't showing.